workspaces/editors.md

```diff
@@ -4,10 +4,11 @@ description:
   Learn how to connect your favorite editors and IDEs to your remote workspace.
 ---
 
-There are five primary ways you can connect an IDE to your Coder workspace:
+There are several primary ways you can connect an IDE to your Coder workspace:
 
 1. [VS Code remote SSH](editors.md#vs-code-remote-ssh) with local VS Code
 1. [VS Code in the browser](editors.md#vs-code-in-the-browser) with code-server
+1. [Local JetBrains and SSH](editors.md#jetbrains-gateway) with JetBrains Gateway
```

Comment:
```suggestion
1. [Local JetBrains and SSH](editors.md#local-jetbrains-and-ssh-with-jetbrains-gateway) with JetBrains Gateway
```

This will make it a valid link